static/styles/settings.css

```diff
@@ -380,6 +380,14 @@ td .button {
     .hidden-email {
         font-style: italic;
     }
+
+    /* Originally the icon inherits the color of label, but when the setting
+    is disabled, the color of the label is changed and thus the icon becomes
+    too light. So, we set the color explicitly to original color of label to
+    keep the color of the icon same even when the setting is disabled. */
+    #id_realm_enable_spectator_access_label a {
+        color: hsl(0, 0%, 20%);
+    }
```

Comment:
Does this need corresponding CSS for the dark theme?

Reply:
Yes, missed it before. Pushed after adding it.

Disabled
![Screenshot from 2021-12-03 00-33-31](https://user-images.githubusercontent.com/35494118/144486728-fa32d5ed-c611-4d5e-accf-e1cd6fcfc54e.png)

Enabled
![Screenshot from 2021-12-03 00-33-14](https://user-images.githubusercontent.com/35494118/144486737-50f7e32c-2cc6-4562-a488-0a3b39fb1630.png)
.